packages/react-components/react-checkbox/src/stories/Checkbox/CheckboxChecked.stories.tsx

```diff
@@ -1,5 +1,5 @@
 import * as React from 'react';
-import { Checkbox, CheckboxProps } from '../../index';
+import { Checkbox, CheckboxProps } from '@fluentui/react-checkbox';
```

Comment:
```suggestion
import { Checkbox } from '@fluentui/react-checkbox';
import type { CheckboxProps } from '@fluentui/react-checkbox';
```
Could this be split like this?